src/package-lock.json

```diff
@@ -1,1260 +1,8 @@
 {
   "name": "almanac.httparchive.org",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Can you revert this file please @kevinfarrugia as unrelated to this PR? Guess you're also on old `npm` :-)

Reply:
Yes, reverted.